bot/exts/backend/auto_config.py

```diff
@@ -0,0 +1,182 @@
+import os
+import typing as t
+import random
+
+from pathlib import Path
+
+import discord
+import yaml
+
+from discord.errors import Forbidden, NotFound
+from discord.ext import commands
+from discord.ext.commands.context import Context
+from loguru import logger
+
+
+class AutoConfig(commands.Cog):
+    """A cog responsible for constructing config files"""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        with open('bot/config/config-list.yaml') as f:
+            self.config_list = yaml.safe_load(f)
+            self.auto = self.config_list['Auto-config']
+            self.config_list.pop('Auto-config')
+
+    @property
+    def guild_id(self) -> int:
+        """ Check's if the guild exists or not, if not then raises NotFound."""
+
+        if not self.bot.get_guild(int(os.getenv('Guild_ID'))):
```

Comment:
```suggestion
        """Check's if the guild exists or not, if not then raises NotFound."""
        if not self.bot.get_guild(int(os.getenv('Guild_ID'))):
```
There shouldn't be a empty space in `def` functions.